fields/types/html/editor/atomic-block-processor.js

```diff
@@ -54,14 +54,15 @@ const processor = {
 				alignment = entity.data && entity.data.alignment || alignment;
 				let caption = _.get(entity, ['data', 'caption'], '');
 				let embeddedCode = _.get(entity, ['data', 'embeddedCode'], '');
-				let script = {};
+				let script;
 				let scripts = [];
 				let scriptTagStart = false;
 				let height;
 				let width;
 				let parser = new htmlparser.Parser({
 					onopentag: (name, attribs) => {
-						if (name === 'script') {
+            if (name === 'script') {
+              script = {};
```

Comment:
fix indent?

Reply:
If we initialize `script={}` here,
we might need to try catch `script.text = text;`,
otherwise, error might happen if `script === undefined`.

Could we clear `script = {}` in the `onclosetag` ?
such as 
```
onclosetag: (tagname) => {
  if (tagname === 'script' && scriptTagStart) {
    scriptTagStart = false;
    scripts.push(_.merge({}, script))
    script = {}
  }
} 
```
?

Reply:
Updated.

Reply:
Updated.